packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -22,10 +24,11 @@ const SIMPLE_SERVER_FILE_PATH = join(__dirname, "simple-server", "bundled_server
 export async function checkBuildConditions(cwd: string): Promise<void> {
   // dynamically load Angular so this can be used in an NPX context
   const { NodeJsAsyncHost }: typeof import("@angular-devkit/core/node") = await import(
-    `${cwd}/node_modules/@angular-devkit/core/node/index.js`
+    require.resolve("@angular-devkit/core/node/index.js")
```

Comment:
You can use the cwd in the second argument, require.resolve(..., { paths: [cwd] }) this will help when invoked from NPX.